### ISSUE TITLE
Settings window issue

### ISSUE THREAD

**Issue by glpzzz** [nuttyartist/notes#613]
![image](https://github.com/nuttyartist/notes/assets/13596248/67c37851-7797-4581-9494-92bc7639d81b)

when clicking in the 3 dots icon, in top right, the app ends up like this. No warnings or errors on terminal. 

Ubuntu 23.04

**Comment by zjeffer** [comment]
Are you running wayland?

**Comment by glpzzz** [comment]
No. I'm running Xorg. Current compositor is picom. Disabling the compositor makes the transparent windows to display all black, no content.

**Comment by zjeffer** [comment]
Which installation method/package did you use?

Can you check the version of the app and the Qt version (gear icon > About, qt version is all the way at the bottom)

**Comment by glpzzz** [comment]
I built it.

App Version: 2.2.0 (Pro Version)
Qt version: 6.4.2 (built with 6.4.2)

**Comment by zjeffer** [comment]
Thanks. When running through the terminal, do you get any errors?

I think you may be missing the dependency for QML. The dependency list in the readme is probably outdated. Although I would expect the compilation to fail if that's the case.

**Comment by zjeffer** [comment]
Maybe you can test out the appimage to see if the problem goes away.

**Comment by glpzzz** [comment]
I'm submitting a PR to add another package in instructions for Ubuntu 23.04 `qml6-module-qtdatavisualization`. Without this one, is not possible to build. 

When running from the terminal, this is the output:

```
QApplication: invalid style override 'kvantum' passed, ignoring it.
	Available styles: Windows, Fusion
qrc:/qt/qml/EditorSettings.qml: module "kvantum" is not installed
qrc:/qt/qml/kanbanMain.qml: module "kvantum" is not installed
Database: connection ok
```

but except for kanban view, it works good.

**Comment by zjeffer** [comment]
What's the output of `echo $QT_QPA_PLATFORMTHEME`?

**Comment by glpzzz** [comment]
blank. I don't have that env var.
```bash
$ env | grep QT                                                                                           
QT_STYLE_OVERRIDE=kvantum
```

**Comment by zjeffer** [comment]
Ah yes, that's the one I meant. 

Looks like overriding the style to one that QML doesn't know about makes the QML windows invisible. We'll fix this. Thanks for reporting.

For now, you can run Notes with `QT_STYLE_OVERRIDE="" notes` until the bug is fixed.

**Comment by nuttyartist** [comment]
@glpzzz So still the editor settings and Kanban view don't work?

**Comment by glpzzz** [comment]
![image](https://github.com/nuttyartist/notes/assets/13596248/9c8b7df4-ebfe-4f0b-bac6-e836f1b5e801)

Thanks! Settings window now works. It seems is not a regular window right? I have not checked the source yet. 

When activating kanban from the settings window, it kind of works, but only shows notes created from kanban mode. Not the others. 

I see there is a __Pro__ icon below kanban. Perhaps this is my issue? Or do building it from source bypasses pro check?

**Comment by zjeffer** [comment]
* It is indeed not supposed to be a regular window, but a popup.
* Kanban mode will only work with a specific format. Create a new item with the + icon in the kanban view, and go back to the regular view to see the format it creates.
* Building it from source gets you the Pro features.

**Comment by nuttyartist** [comment]
> I see there is a Pro icon below kanban. Perhaps this is my issue? Or do building it from source bypasses pro check?

It bypasses it. I forgot to remove the Pro label for Pro versio, good you reminded me.

The rest answered by @zjeffer  👍

**Comment by glpzzz** [comment]
Thaks! Good app!

**Comment by guihkx** [comment]
> When running from the terminal, this is the output:
> 
> ```
> QApplication: invalid style override 'kvantum' passed, ignoring it.
> 	Available styles: Windows, Fusion
> qrc:/qt/qml/EditorSettings.qml: module "kvantum" is not installed
> qrc:/qt/qml/kanbanMain.qml: module "kvantum" is not installed
> Database: connection ok
> ```
> 
> but except for settings windows and kanban view, it works good.

Looks like this issue (which is about our Flatpak package, but the symptom is the same):

- https://github.com/flathub/io.github.nuttyartist.notes/issues/1

Last time I researched about this bug, it led me to [this Qt bug](https://bugreports.qt.io/browse/QTBUG-99889), which has no solution yet. We only have the workaround already mentioned here by @zjeffer.
